.gitignore

```diff
@@ -3,6 +3,7 @@ workspace
 .metadata/*
 .recommenders/*
 .DS_Store
+.project
```

Comment:
The project files are actually check-in (common praxis with Eclipse) so that everybody gets the same configuration.

Reply:
1e8f5508f824617cc6eb495c2ea5ab54c13783c6 thanks!